e2e/tests/catalogue/reset-page-on-filter-change.spec.ts

```diff
@@ -8,7 +8,7 @@ test.beforeEach(async ({ context, baseURL }) => {
 test('offset should be reset ( back to page 1) when filter changes ', async ({ page }) => {
   await page.goto('/catalogue-demo/ssr-catalogue/EUChildNetwork/variables');
   await expect(page.locator('text=abd_circum_sdsWHO_t').first()).toBeVisible();
-  await page.locator('nav').filter({ hasText: 'Page OF' }).getByRole('button').nth(1).click();
+  await page.locator('a').filter({ hasText: 'Go to page 2' }).click();
```

Comment:
Previously, the selector pointed to an svg element as there wasn't a label that describes the action of the button. In this PR, I added a label to the buttons. When I re-recorded the test, the path to the text element is used rather than the svg element.